common/src/domain/pokemon/pokemon.ts

```diff
@@ -22,7 +22,7 @@ export interface Pokemon {
   skillPercentage: number;
   berry: Berry;
   carrySize: number;
-  maxCarrySize: number;
```

Comment:
The old frontend uses this to automatically select the default skill level. Unfortunately since the old site is ass it doesn't give compile errors. I just tested the site by checking out branch and running a production calc, and it wouldn't let me calc because skill level wasn't set.

We need to change this line in functions.js line 320:
`var nrOfEvolutions = (pokemonInfo.maxCarrySize - pokemonInfo.carrySize) / 5;`
To this:
`var nrOfEvolutions = pokemonInfo.previousEvolutions;`

Reply:
Done